source/_integrations/analytics_insights.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Home Assistant Analytics Insights
+description: Instructions on how to integrate Home Assistant Analytics Insights within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: '2024.2'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@joostlek'
+ha_domain: analytics_insights
+ha_platforms:
+  - sensor
+ha_integration_type: service
+---
+
+The **Analytics Insights** {% term integration %} allows you to get integration usage statistics into Home Assistant.
```

Comment:
I would add a link to https://analytics.home-assistant.io/ where the data comes from